stdlib/src/builtin/string.mojo

```diff
@@ -361,10 +361,10 @@ fn isupper(c: Int8) -> Bool:
     Returns:
         True if the character is uppercase.
     """
-    return _is_ascii_uppercase(c)
+    return _is_ASCII_UPPERCASE(c)
```

Comment:
What is the purpose to break consistency and not to keep snake_case? This is internal functionality. There is no need to keep this aligned with Python. Same to below

Reply:
That was accidental by using `change all occurences` instead of normal renaming in my IDE.
Good catch, thx!